libs/genai/langchain_google_genai/chat_models.py

```diff
@@ -419,6 +419,8 @@ def _parse_response_candidate(
     for part in response_candidate.content.parts:
         try:
             text: Optional[str] = part.text
+            # Remove erroneous newline character if present
+            text = text.rstrip("\n")
```

Comment:
if text can be None, we should check for this before applying `rstrip`

Reply:
I've added a conditional for this.